3p/iframe-messaging-client.js

```diff
@@ -107,9 +110,6 @@ export class IframeMessagingClient {
 
   /**
    * Register callback function for message with type messageType.
-   *   As it stands right now, only one callback can exist at a time.
```

Comment:
is it not the case anymore?

Reply:
It was no longer the case after #8591, but the comment persists. Cleaning it up while I'm here.